fire/SFMainMod.F90

```diff
@@ -668,6 +667,7 @@ subroutine  area_burnt_intensity ( currentSite )
     use FatesConstantsMod, only : years_per_day
     use SFParamsMod,       only : SF_val_fdi_alpha,SF_val_fuel_energy, &
          SF_val_max_durat, SF_val_durat_slope
+    use SFParamsMod, only  : SF_val_fire_threshold
```

Comment:
```suggestion
```

Reply:
github's resolve feature did not get this fix correct, will apply manually